vllm/distributed/kv_transfer/kv_connector/v1/nixl_connector.py

```diff
@@ -382,12 +379,8 @@ def request_finished(
                 or request.status != RequestStatus.FINISHED_LENGTH_CAPPED):
             return False, None
 
-        # Get computed blocks.
-        all_full = request.num_computed_tokens % self.block_size == 0
-        computed_block_ids = block_ids if all_full else block_ids[:-1]
-
         # If prompt < block_size, no xfer so free blocks immediately.
-        delay_free_blocks = len(computed_block_ids) > 0
+        delay_free_blocks = len(block_ids) > 0
```

Comment:
I wonder would `block_ids` actually ever be 0 here? If not we could remove the conditional below